CMakeLists.txt

```
@@ -174,6 +173,11 @@ if (NOT BUILD_TESTS_ONLY)
   set(THREADS_PREFER_PTHREAD_FLAG TRUE)
   find_package(Threads REQUIRED)
 
+  include(cmake/find_pmix.cmake)
+  check_pmix()
+
+configure_file(cmake/rocshmem_config.h.in rocshmem_config.h)
```

Picky comment but indent:

```suggestion
  configure_file(cmake/rocshmem_config.h.in rocshmem_config.h)
```